### PR TITLE
CHAT-1066: fixing unread message bug

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
(ID task=23604)
- In order to fix the problem of unread message, we have no need to store selected contact in local storage in exo chat drawer, and we remove old used method in min chat `windowFocused() `and `minimized()` in `ExoChatMessageList.vue`.Although we invoke the 2 methods `markRoomMessagesRead() `and` contactChanged() `only when you selected a contact not from the start of chat application

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 1]
```diff
@@ -317,6 +319,8 @@ export default {
     },
     backChat(){
       this.listOfContact=true;
+      this.selectedContact = [];
+      this.listOfContact = true;
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:323]: duplicated instruction.

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 2]
```diff
@@ -290,6 +291,7 @@ export default {
       window.location.reload();
     },
     openRoom(e) {
+      document.body.style.overflow = 'hidden';
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:294]: Is this needed for the current fix ?

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 1 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:323]: ok,sorry

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 2 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:294]: no i will remove it

**Pull request merged**